#!/usr/bin/env python
PACKAGE = "aerial_robot_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_depth",                double_t, 0, "min depth [m]",            0.3,  0.0, 5.0)
gen.add("max_depth",                double_t, 0, "max depth [m]",            2.0,  0.0, 10.0)
gen.add("distance_from_ground",     double_t, 0, "distance from ground [m]", 0.1,  0.0, 1.0)
gen.add("use_distance_from_ground", bool_t,   0, "use distance from ground", True)
gen.add("use_otsu_binarization", bool_t,   0, "use otsu binarization", False)

exit(gen.generate(PACKAGE, "aerial_robot_perception", "DepthDistanceFilter"))
